### PR TITLE
Add zsh completion for '--ipam-driver --subnet --ip-range --gateway -…

### PR DESCRIPTION
**Pull request by sdurrheimer**
…-aux-address' for 'docker network create'

Ref. #16910 

@vincentbernat 

Signed-off-by: Steve Durrheimer s.durrheimer@gmail.com

### CODE REVIEW

**Comment by tiborvass** [comment]
LGTM

**Comment by vincentbernat** [comment]
LGTM

File: contrib/completion/zsh/_docker [hunk 1]
```diff
@@ -253,6 +253,11 @@ __docker_network_subcommand() {
             _arguments -A '-*' \
                 $opts_help \
                 "($help -d --driver)"{-d,--driver=}"[Driver to manage the Network]:driver:(null host bridge overlay)" \
+                "($help)--ipam-driver=-[IP Address Management Driver]:driver:(default)" \
+                "($help)*--subnet=-[Subnet in CIDR format that represents a network segment]:IP/mask: " \
+                "($help)*--ip-range=-[Allocate container ip from a sub-range]:IP/mask: " \
+                "($help)*--gateway=-[ipv4 or ipv6 Gateway for the master subnet]:IP: " \
+                "($help)*--aux-address[Auxiliary ipv4 or ipv6 addresses used by network driver]:key=IP: " \
```
Comment (albers) [inline contrib/completion/zsh/_docker]: @sdurrheimer There is an additional option `--opt|-o`, see https://github.com/docker/docker/pull/17046/files#diff-37d5f4118ca6b64624af89485f9a0f61R49

Comment (sdurrheimer) [inline contrib/completion/zsh/_docker]: Yep, that PR wasn't yet merged when I've done this.

**Pull request merged**